src/modules/portfolio/containers/transactions.js

```diff
@@ -1,16 +1,24 @@
 import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
-
+import { loadAccountHistory } from 'modules/auth/actions/load-account-history'
+import { selectTransactions } from 'modules/transactions/selectors/transactions'
 import TransactionsList from 'modules/portfolio/components/transactions/transactions'
 
 const mapStateToProps = state => ({
-  transactions: dummyTransactions
+  transactions: selectTransactions(state),
+  transactionsLoading: state.transactionsLoading,
+})
+
+const mapDispatchToProps = dispatch => ({
+  // TODO: need to get parameters from configuation?
```

Comment:
How I would do this is have this method consume some args which are passed via a call make w/in the component.

Reply:
Such as on the change of a select element — call this method w/ the selected option.

Reply:
How do you want to cascade changes to date filter dropdown?
global event? or is there a cool react way for component to component comm?

Reply:
The date dropdown is in performance-graph.jsx, I'm not sure how transactions.jsx would get it's state. I don't see a parent-child relationship?